src/model.rs

```diff
@@ -337,15 +340,23 @@ impl Model<PluginsIncluded> {
         self.scip
             .create_prob(name)
             .expect("Failed to create problem in state PluginsIncluded");
-        self.move_to_state::<ProblemCreated>()
+        Model {
+            scip: mem::take(&mut self.scip),
```

Comment:
This calls ScipPtr::default, instantiating a new scip problem. Probably not what you wanted.

Reply:
Ah yes! fixed it.